### ISSUE TITLE
Missing data for platform, chassis id etc.,

### ISSUE THREAD

**Issue by f5-mshan** [F5Networks/f5-telemetry-streaming#26]
##### ISSUE TYPE
 - Bug Report

##### Telemetry Streaming BUILD/ VERSION
``` 
1.7.0.1 

```

##### BIGIP VERSION

```
[root@xyz:Active:Standalone] ~ # tmsh show sys version

Sys::Version
Main Package
  Product     BIG-IP
  Version     14.1.0.6
  Build       0.0.9
  Edition     Point Release 6
  Date        Fri Jun 21 21:06:44 PDT 2019

[root@xyz:Active:Standalone] ~ # 

```

##### SUMMARY

```
When the system poller is enabled, it is missing the following information,

1. marketingName
2. platformId
3. chassisId


```

##### STEPS TO REPRODUCE

```
1. Configure a poller/consumer using the below config,

"Poller": {
    "class": "Telemetry_System_Poller",
    "interval": 60,
    "enable": true,
    "trace": false,
    "allowSelfSignedCert": false,
    "host": "localhost",
    "port": 8100,
    "protocol": "http"
  }
"Consumer": {
    "class": "Telemetry_Consumer",
    "type": "Generic_HTTP",
    "host": "<host>",
    "protocol": "https",
    "port": 50443,
    "path": "<path>",
    "method": "POST",
    "enable": true,
    "trace": true,
 },

2. Check the trace output and notice the missing platform information

system": {
            "hostname": "<hostname>",
            "version": "14.1.0.6",
            "versionBuild": "0.0.9",
            "location": "missing data", 
            "description": "missing data",
            "marketingName": "missing data",
            "platformId": "missing data",       ---> Notice the missing data
            "chassisId": "missing data",
            "baseMac": "<basemac>",
            "callBackUrl": "<callbackurl>",
            "configReady": "yes",
            "licenseReady": "yes",
            "provisionReady": "yes",
            "syncMode": "standalone",
            "syncColor": "green",
            "syncStatus": "Standalone",
            "syncSummary": " ",
            "failoverStatus": "ACTIVE",
            "failoverColor": "green",
            "systemTimestamp": "2019-10-29T19:11:54.000Z",
            "cpu": 3,
            "memory": 20,
            "tmmCpu": 1,

```

##### ACTUAL RESULTS

system": {
            "hostname": "<hostname>",
            "version": "14.1.0.6",
            "versionBuild": "0.0.9",
            "location": "missing data",
            "description": "missing data",
            "marketingName": "missing data",
            "platformId": "missing data",
            "chassisId": "missing data",
            "baseMac": "<basemac>",
            "callBackUrl": "<callbackurl>",
            "configReady": "yes",
            "licenseReady": "yes",
            "provisionReady": "yes",
            "syncMode": "standalone",
            "syncColor": "green",
            "syncStatus": "Standalone",
            "syncSummary": " ",
            "failoverStatus": "ACTIVE",
            "failoverColor": "green",
            "systemTimestamp": "2019-10-29T19:11:54.000Z",
            "cpu": 3,
            "memory": 20,
            "tmmCpu": 1,

```

```

**Comment by dstokesf5** [comment]
I have added this to our internal product backlog as AUTOTOOL-896. Thank you for your feedback.

**Comment by dstokesf5** [comment]
This was resolved as part of the 1.10.0 release. Closing this issue. If you are still experiencing a problem, please reopen this issue or create a new one.
